resource-service/src/main/java/uk/gov/gchq/palisade/service/resource/request/ResourceRequest.java

```diff
@@ -44,13 +44,16 @@
 
     // Unique identifier for the user
     private final String userId;
-    // Resource Id of the resource requested
-    public final String resourceId;
     // Json Node representation of Context
     private final JsonNode context;
     // Json Node representation of the User
     private final JsonNode user;
 
+    /**
+     * Resource Id of the resource requested
+     */
+    public final String resourceId;
+
```

Comment:
SonarQube?

Reply:
Yep. Became a smell after a develop merge, somehow. Thought I'd just fix it.